packages/fxa-settings/src/pages/Signin/container.tsx

```diff
@@ -424,9 +438,33 @@ const SigninContainer = ({
         }
       }
 
+      // Send totp token email if session is not verified at this point since users will
+      // be redirected to /signin_token_code
```

Comment:
Also, this left me wondering why we don't just automatically send a code on that page since we seem to manage it beforehand. Maybe we don't want refreshes to trigger a new code being sent?